.github/workflows/ci.yml

```diff
@@ -32,12 +46,25 @@ jobs:
         uses: ./
 
       - name: Test Toit in Path
+        if: ${{ matrix.is_legacy != true }}
```

Comment:
I think you need to check against `'true'` (the string). But not 100% sure.

Reply:
I can explicitly also set `is_legacy: 'true'` and `${{ matrix.is_legacy != 'true' }}` however if that is preferred?